package.json

```diff
@@ -17,6 +17,8 @@
     "@types/react-dom": "^16.9.10",
     "@types/react-fontawesome": "^1.6.4",
     "@types/react-inspector": "^4.0.1",
+    "@types/react-virtualized-auto-sizer": "^1.0.1",
```

Comment:
Do I need to bump the version manually or does this happen upon release?

- [ ] Check before merging